cmake/configure/configure_2_trilinos.cmake

```diff
@@ -281,6 +281,43 @@ MACRO(FEATURE_TRILINOS_FIND_EXTERNAL var)
       ENDIF()
     ENDIF()
 
+    # the only thing we use from SEACAS right now is ExodusII, so just check
+    # that it works
+    IF(${DEAL_II_TRILINOS_WITH_SEACAS})
+      LIST(APPEND CMAKE_REQUIRED_INCLUDES ${Trilinos_INCLUDE_DIRS})
+      LIST(APPEND CMAKE_REQUIRED_LIBRARIES ${Trilinos_LIBRARIES})
+      CHECK_CXX_SOURCE_COMPILES(
+        "
+        #include <exodusII.h>
+        int
+        main()
+        {
+          int component_word_size = sizeof(double);
+          int floating_point_word_size = 0;
+          float ex_version = 0;
+          const int ex_id = ex_open(\"test.ex\",
```

Comment:
I wished Trilinos would pay more attention to namespacing :-(

Reply:
To be fair Exodus, AFAICT, very much predates Trilinos (some of the technical documents are from the early 1990s).